README.md

```diff
@@ -1,4 +1,6 @@
 # Igbo Dictionary API
+![Travis (.org)](https://img.shields.io/travis/ijemmao/igbo_api)
```

Comment:
It seems like the current build stat is 'unknown', but it should be 'success' since the latest commit has a green checkmark.

Were you able to produce a success status badge?

Reply:
Sorry for the late response.
Yes you're right, the current build stat is being displayed as 'unknown'. Though I've realized the mistake now and will be fixing it soon